src/scss/bootstrap3_compatibility.scss

```diff
@@ -0,0 +1,848 @@
+/*
+  This file can be used to put any scss rules to fix Bootstrap v3 to v4 issues.
```

Comment:
@safydy This is duplicated from `voto5` repo without changes, right?

Reply:
yes, that's right